shared/constants/dates.ts

```diff
@@ -0,0 +1,3 @@
+export const DATE_DISPLAY_FORMAT = 'dd/MM/yyyy'
```

Comment:
Hmm actually our display and parse format is the same right? Is there a reason why we continue to keep them separated?

Reply:
They're currently the same, but as they have distinct meanings I think it's better to keep them separate. One example [here](https://github.com/opengovsg/FormSG/blob/develop/frontend/src/templates/Field/Date/DateField.tsx#L84-L85)

Reply:
Ahh I see okay thanks @wanlingt, and thanks for abstracting them out to a shared constants file